packages/fxa-shared/email/validateEmail.js

```diff
@@ -0,0 +1,32 @@
+const dns = require('dns');
+const resolver = new dns.promises.Resolver();
+const NotFoundErrorCodes = require('./emailValidatorErrors').NotFoundErrorCodes;
+
+export const results = ['MX', 'A', 'none', 'skip'].reduce(
```

Comment:
What kind of results are these? This is pretty hard to follow.

Reply:
since i dont need this in the auth server i am moving it back to where it came from and leaving the original logic as is